src/NServiceBus.Persistence.ServiceFabric.Tests/ComponentTests/OutboxStorageTests.cs

```diff
@@ -64,7 +65,7 @@ class OutboxStorageTests
                 await transaction.Commit();
             }
 
-            CleanOlderThan(storage, DateTime.UtcNow);
+            await CleanOlderThan(storage, DateTimeOffset.UtcNow);
```

Comment:
I don't know if we have a policy around this, but date time offset is the more accurate implementation when it comes to timezones